client/src/components/Form/components/formSections/BehandlerSection.tsx

```diff
@@ -12,6 +12,7 @@ import SykmelderInformation from '../formComponents/SykmelderInformation';
 import { FormType } from '../../Form';
 import { Section } from '../../../../types/Section';
 import { Sykmelder } from '../../../../types/Sykmelder';
+import moment from "moment";
```

Comment:
bruk `dayjs`, som allerede finnes i appen. Samme API som moment, men mye mindre bibliotek!
```suggestion
import dayjs from "dayjs";
```